core/build.sbt

```diff
@@ -443,7 +443,24 @@ def cloudflowModule(moduleID: String): Project = {
     .enablePlugins(AutomateHeaderPlugin)
 }
 
-lazy val commonSettings = Seq(
+// These settings are made active only when we use bintray for internal release
+// It is important that when we do final releases we need to invoke sbt as 
+// `sbt -Dsbt.sbtbintray=false`
+lazy val bintraySettings =
+  if (BintrayPlugin.isEnabledViaProp) {
+    Seq(bintrayOrganization := Some("lightbend"),
+      bintrayRepository := altBintrayRepository.value.getOrElse("cloudflow"),
+      bintrayOmitLicense := true,
+      publishMavenStyle := false,
+      resolvers += "Akka Snapshots" at "https://repo.akka.io/snapshots/",
```

Comment:
Could combine this line and the line below.

Reply:
@olofwalker On a second look, I am thinking if we really need this akka snapshot repo added to the resolver ? Do we use snapshots ? /cc  @RayRoestenburg

Reply:
done ..